tck/src/main/java/org/eclipse/microprofile/lra/tck/service/LRAMetricType.java

```diff
@@ -27,8 +27,10 @@ public enum LRAMetricType {
     FailedToCancel, // an LRA that failed to cancel
 
     // Participant statistics
-    Compensated, // a participant that was asked to compensate
-    Completed, // a participant that was asked to complete
+    Compensating, // a participant that was asked to compensate
+    Completing, // a participant that was asked to complete
```

Comment:
These new enum values don't seem to be used.

Reply:
The completing one is now used here: https://github.com/microprofile/microprofile-lra/pull/368/files#diff-e547d1e228cde545e98971313c9216d09127f9e26176870910ae796172ca67f0R116
I think those 2 extra statuses are important to have better clarify what the Participant statuses can be

Reply:
I still don't understand what is the value of these intermediate stages.  @marcosgopen please explain when they would be useful as you can see above I'm arguing that the Status method shouldn't be invoked before Complete/Compensate.

Reply:
If this is your concern, even if the implementation of the Complete/Compensate basically return immediately, feel free to change the return values to `CompletableFuture.completedFuture`.